pkg/model/job.go

```diff
@@ -215,6 +215,8 @@ type JobSpecDocker struct {
 	Image string `json:"Image,omitempty"`
 	// optionally override the default entrypoint
 	Entrypoint []string `json:"Entrypoint,omitempty"`
+	// Parameters holds the default arguments that are passed to the ENTRYPOINT when the container is run.
```

Comment:
I think this comment is wrong? It's the "extra arguments that are passed on the commandline when the container is run", no?